registry/refresh.js

```diff
@@ -17,11 +17,13 @@ function isDeprecated(typeName) {
 
 async function main() {
   let nextToken = undefined;
+  await fs.rmdir(outdir, { recursive: true })
+  await fs.mkdir(outdir);
```

Comment:
To remove types existing types we decided to ignore. Otherwise, the existing version of the type will remain forever. 

Note that this caused the removal of currently deprecated resources.